main.go

```diff
@@ -0,0 +1,21 @@
+// servedot command start http server, serving files in current directory
+//       go run foxygo.at/servedot
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+)
+
+func main() {
+	http.Handle("/", http.FileServer(http.Dir(".")))
+	listener, err := net.Listen("tcp", "localhost:0")
```

Comment:
it would be handy to be able to specify a port. If you restart the server, the port will change and if you are using command line history for the command at the other end, you'll need to keep changing that too.

Perhaps version 2.

Reply:
ok, great point.
I will do this in the next PR, together with arg for `dir` and maybe some CI (or maybe separate PRs ;) )